Xamarin.Forms.Core/OnAppTheme.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+
+namespace Xamarin.Forms
+{
+	public class OnAppTheme<T> : BindableObject, IDisposable
```

Comment:
not a BO. look at OnPlatform, OnDevice

Reply:
also, you might want a xaml markup language extension

Reply:
Because I'm using bindable properties? Should I be using something different? And there is an extension as well; https://github.com/xamarin/Xamarin.Forms/pull/9958/files#diff-c65a7e8e0eed508c58cd0802b61b2cb1